### ISSUE TITLE
dartaotruntime is missing from Linux .deb package

### ISSUE THREAD

**Issue by cpswan** [dart-lang/sdk#49293]
* Dart SDK Version `Dart SDK version: 2.17.3 (stable) (Unknown timestamp) on "linux_x64"`

If I install Dart from `dart_2.17.3-1_amd64.deb` then `dartaotruntime` is missing.

Whilst if I install it from `dartsdk-linux-x64-release.zip` it's there in `dart-sdk/bin/dartaotruntime`

**Comment by whesse** [comment]
I cannot reproduce this.  If I delete my entire dart installation, and then install it using

    sudo dpkg -i dart_2.17.3-1_amd64.deb

then I do get the dartaotruntime executable in /usr/lib/dart/bin/dartaotruntime.

**Comment by cpswan** [comment]
Yep. The file is indeed there, and it seems the problem is that `/usr/bin/dart` is symlinked to `../lib/dart/bin/dart` whilst there's no such symlink for `dartaotruntime` and `/usr/lib/dart/bin/` isn't on the PATH.

**Comment by whesse** [comment]
@mit-mit We can consider if adding a symlink to that runtime library is appropriate.  I'm not sure what the distribution plan is for executable compiled Dart programs.  Just making them work locally by putting the runtime on the PATH is not enough to let them also be distributed.

**Comment by rmacnak-google** [comment]
dartaotruntime is on the PATH as of 3.9.0
